Server/controllers/comments.controller.js

```diff
@@ -9,4 +9,25 @@ exports.getCommentsByArticleID = (req, res, next) => {
     res.status(200).send( { comments } );
     })
     .catch( err => { next(err) })
+};
+
+exports.postCommentByArticleID = (req, res, next) => {
+
+    const article_id  = req.params.article_id
+    const userName = req.body.username
+    const commentBody = req.body.body
+
+    if (userName === undefined) {
+        return next({ status: 400, msg: "incomplete request, provide username" })
+    }
+
+    if(commentBody === undefined) {
+        return next({ status: 400, msg: "incomplete request, please add a comment" }) 
+    }
```

Comment:
These two conditions are not necessary, if you are using the `$` binding syntax in your query then PSQL will throw an error for you if either of these values are undefined